Apps/DBMaintainer/appsettings.Development.json

```diff
@@ -12,27 +12,27 @@
     "FedApprovedDatabase": {
         "Url": "https://www.canada.ca/content/dam/hc-sc/documents/services/drug-product-database/allfiles_ap.zip",
         "TargetFolder": "Resources",
-        "DBCode":  "105"
+        "AppName":  "FAPP"
```

Comment:
nit: is it necessary for the codes to be that short? can we make them more verbose for readability?

Reply:
I made the length 10 but I couldn't think of anything better :)  Maybe FEDDB-APR?

Reply:
Updated

Reply:
What about FedDatabase?